src/db.ts

```diff
@@ -358,6 +363,19 @@ export class Database {
           }
         }
 
+        const upstreamAuthors = await this.get(
+          "SELECT * FROM pragma_table_info('pending_prs') WHERE name='upstream_authors'",
+        );
+        if (!upstreamAuthors) {
+          await this.run(
+            "ALTER TABLE pending_prs ADD upstream_authors TEXT DEFAULT 'not-applicable' NOT NULL",
```

Comment:
After thinking about this, I think rather than set "TEXT DEFAULT", I would add the column as nullable, set all existing rows to have the value `not-applicable`, and then alter the column to have the not null constraint.

The reason being that I think if you just don't provide a value, it'll be set to `not-applicable`.

Reply:
This SQL thing is still new to me, but I followed some commands in this blog: https://www.sqlservercentral.com/articles/nullable-vs-non-nullable-columns-and-adding-not-null-without-downtime-in-postgresql

Let me know whether it looks right. 🤷‍♂️